test/hotspot/jtreg/runtime/cds/appcds/StaticArchiveWithLambda.java

```diff
@@ -45,11 +44,14 @@ public static void main(String[] args) throws Exception {
         String archiveName = "StaticArchiveWithLambda.jsa";
 
         // dump class list
-        ProcessBuilder pb = ProcessTools.createTestJvm(
-            "-XX:DumpLoadedClassList=" + classList,
-            "-cp", appJar,
-            appClass);
-        OutputAnalyzer output = TestCommon.executeAndLog(pb, "dumpClassList");
+        CDSOptions dumpOpts = (new CDSOptions())
+            .setUseVersion(false)
+            .setXShareMode("auto")
+            .addSuffix("-XX:DumpLoadedClassList=" + classList,
+                       "-cp", appJar,
+                       appClass);
+        CDSTestUtils.run(dumpOpts)
+                    .assertNormalExit();
```

Comment:
I see several places with similar code. Maybe the above can be moved into a common method like

```
CDSTestUtils.dumpClassList(classlist, "-cp", appJar,  appClass);
```

`CDSTestUtils.dumpClassList` can internally call `assertNormaExit`, and return the `Result` so the test case can perform more tests if necessary.

Reply:
Hi Ioi,

Thanks for your review!
I've incorporated all your comments in the 01 version of webrev.

thanks,
Calvin